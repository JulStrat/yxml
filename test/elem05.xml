<:A_S0mewhat-Longer.Name></:A_S0mewhat-Longer.Name>
