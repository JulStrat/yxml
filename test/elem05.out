
elemstart :A_S0mewhat-Longer.Name
elemend
ok
